### ISSUE TITLE
the input size of training net

### ISSUE THREAD

**Issue by mst9009** [zhaoweicai/mscnn#26]
HI @zhaoweicai 
According to the caffe code you provided , I wonder that if the crop_width and crop_height in image_gt_data_param determined the input size of training net, if so , the input size of training net should be different from the net in deploy file because there is no crop in deploy, so how it works correctly?
Thank you!

**Comment by zhaoweicai** [comment]
hi @MST9009 
Please read fine-tuning in Sec. 3.4 in the paper. crop_width and crop_height decide the training input size. I think there is no need to train on the whole image for KITTI. It is just a waste of memory and time. It works very well for me. You can give it a try to train on the whole image and see the difference.

**Comment by mst9009** [comment]
hi @zhaoweicai 
Thanks for your reply.
I've learnt the benefits of crop. to my understanding of the code ,the procedure of crop seem to be determined by parameters "min_scale" and "max_scale", the "crop_width" and "crop_height" seem to just change the template size of training net, if so, is it necessary to change the input dim of images to the crop-size rather than the resize-size in practical application.
Thank you!
